lib/membrane_element_sdl/player.ex

```diff
@@ -12,23 +12,15 @@ defmodule Membrane.Element.SDL.Player do
   use Membrane.Sink
   use Bunch
 
-  @experimental_latency 20 |> Time.milliseconds()
-
-  def_options latency: [
-                type: :time,
-                default: @experimental_latency,
-                description: """
-                Time needed to show a frame on a screen.
-                May have to be adjusted for your system.
-                """
-              ]
+  # Experimentally designated time needed to show a frame on a screeen
```

Comment:
designated doesn't seem like the right word here. I would rephrase it as: 
```suggestion
  # The latency needed to show a frame on a screen measured in an experiment
```